fec.config.js

```diff
@@ -1,13 +1,121 @@
+const { resolve } = require('path');
+const packageJson = require('./package.json');
+const webpack = require('webpack');
+
+const bundle = 'insights';
+const appName = packageJson[bundle].appname;
+
 module.exports = {
-  appUrl: '/staging/starter',
+  appName,
+  appUrl: `/${bundle}/${appName}`,
+  useProxy: process.env.PROXY === 'true',
   debug: true,
-  useProxy: true,
-  proxyVerbose: true,
-  /**
-   * Change to false after your app is registered in configuration files
-   */
-  interceptChromeConfig: false,
-  /**
-   * Add additional webpack plugins
-   */
+  plugins: [
+    new webpack.DefinePlugin({
+      IS_DEV: false,
```

Comment:
I would set this boolean value to `process.env.NODE_ENV === 'development'`. Because otherwise with your change there will be never IS_DEV=true when working for example locally on stage where we would need redux logger and some other enhancers enabled in dev environment.

Reply:
+1 However, I think in the future we should replace this. It's used to add or not add the `redux-logger`, We can do that differently, similar to how we do it elsewhere.